defaults/main.yml

```diff
@@ -8,5 +8,7 @@ slapd_olc_config:
   dummy: value
 slapd_monitor_config:
   dummy: value
+slapd_frontend_config:
+  dummy: value
```

Comment:
Can't you check if it is defined when using it? Or try `{}`, this works for me in another role.